app/views/components/answers.scala.html

```diff
@@ -19,21 +19,21 @@
 
 @(answers: Seq[Answer], horizontal: Boolean, name: String)(implicit request: Request[_], messages: Messages)
 
-@answerText(answer: Answer) = @{ answer.txt }
+@answerText(answer: Answer) = @{ answer.text }
 
 <div class="govuk-radios@{if(horizontal) " govuk-radios--inline" else ""}">
   @for((ans, index) <- answers.zipWithIndex) {
 
     <div class="govuk-radios__item">
       <input class="govuk-radios__input"
             id="@name-item-@index"
-            name="questionPageUrl"
+            name="url"
```

Comment:
Is there any particular reason for using Url here? In the UK gov style guide the id and name are assigned the same value.

Reply:
Yes, good spot, I'd come across also while working on current ticket, will correct here also